src/LogEntryInt.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef LOGENTRYINT_HPP
+#define LOGENTRYINT_HPP
+
+#include <iostream>
+#include <string>
+
+class LogEntryInt
```

Comment:
The `Int` suffix might be a bit misleading :P 

The most common way of indicating something is an interface is to add `I` prefix - `ILogEntry`